Lib/ufo2ft/postProcessor.py

```diff
@@ -357,6 +361,99 @@ def _subroutinize_with_cffsubr(cls, otf, cffVersion):
 
         return cffsubr.subroutinize(otf, cff_version=cffVersion, keep_glyph_names=False)
 
+    def apply_fontinfo(self):
+        """Apply the fontinfo data from the DesignSpace variable-font's lib to
+        the compiled font."""
+        import copy
+
+        # Create a temporary UFO and sets its fontinfo to the union of the main
+        # UFO's fontinfo and the DesignSpace variable-font’s info.
+        temp_ufo = type(self.ufo)()
+        temp_ufo.info = copy.copy(self.ufo.info)
+        for k, v in self.info.items():
+            setattr(temp_ufo.info, k, v)
+
+        # Build a temporary font with the only tables that can be modified with
+        # fontinfo.
+        tables = {"head", "hhea", "name", "OS/2", "post"} & set(self.otf.keys())
+        compiler = InfoCompiler(
+            temp_ufo,
+            glyphSet={},
+            glyphOrder=[],
+            tables=tables,
+        )
+        temp_otf = compiler.compile()
+
+        # Merge the modified data from the temporary font to the main font.
+        for tag in tables:
+            temp = temp_otf[tag]
+            orig = self.otf[tag]
+            if tag == "name":
+                temp_names = {
+                    (n.nameID, n.platformID, n.platEncID, n.langID): n
+                    for n in temp.names
+                }
+                orig_names = {
+                    (n.nameID, n.platformID, n.platEncID, n.langID): n
+                    for n in orig.names
+                }
+                orig_names.update(temp_names)
+                orig.names = list(orig_names.values())
+                continue
+
+            for attr in temp.__dict__:
```

Comment:
instead of negative list of attributes to exclude/continue, I think it would be clearer/less error prone if we had a (positive) list of all the attributes that we do support and and can be overridden this way